Shared/Samples/Get elevation at point on surface/GetElevationAtPointOnSurfaceView.swift

```diff
@@ -0,0 +1,131 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct GetElevationAtPointOnSurfaceView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// The location callout placement.
+    @State private var calloutPlacement: CalloutPlacement?
+    
+    /// The screen point where to get the base surface location.
+    @State private var screenPoint: CGPoint!
+    
+    /// The scene point where the scene was tapped.
+    @State private var scenePoint: Point!
+    
+    /// The surface elevation of the tapped point.
+    @State private var elevation: Double?
+    
+    var body: some View {
+        SceneViewReader { sceneViewProxy in
+            SceneView(scene: model.scene)
+                .onSingleTapGesture { screenPoint, scenePoint in
+                    self.screenPoint = screenPoint
+                    self.scenePoint = scenePoint
+                }
+                .callout(placement: $calloutPlacement.animation(.default.speed(2))) { _ in
+                    VStack(alignment: .leading) {
+                        Text("Elevation")
+                            .font(.headline)
+                        Text(model.elevationString(elevation: elevation))
+                            .font(.callout)
+                    }
+                    .padding(5)
+                }
+                .task(id: scenePoint) {
+                    guard let scenePoint,
+                          // Converts the tapped screen point into a point on the surface.
+                          let relativeSurfacePoint = sceneViewProxy.baseSurfaceLocation(fromScreenPoint: screenPoint) else { return }
+                    if calloutPlacement == nil {
+                        // Gets the elevation from the tap location.
+                        elevation = await elevation(at: relativeSurfacePoint)
+                        // Shows the callout at the tapped location.
+                        calloutPlacement = CalloutPlacement.location(scenePoint)
+                    } else {
+                        // Hides the callout.
+                        calloutPlacement = nil
+                    }
+                }
+        }
+    }
+    
+    /// Gets the elevation at the surface point.
+    /// - Parameter surfacePoint: A geographical location on the base surface
+    /// in the same spatial reference as the scene view.
+    /// - Returns: The elevation in meters.
+    private func elevation(at surfacePoint: Point) async -> Double? {
```

Comment:
This method is separated from the `.task` on purpose, even if it only has 1 line. It's intended for the user to find the key aspect of this sample quickly. 

Since it is unlikely to have an error when the device is connected, the error handling is omitted. This is a deviated behavior from the design's implementation details.